packages/pds/src/db/index.ts

```diff
@@ -55,20 +58,61 @@ export class Database {
       dialect: new PostgresDialect({ pool }),
     })
 
-    return new Database(db, 'pg', schema)
+    return new Database(db, { dialect: 'pg', pool, schema })
   }
 
   static memory(): Database {
     return Database.sqlite(':memory:')
   }
 
+  notify(channel: string, msg?: string) {
+    if (this.facets.dialect === 'pg') {
+      this.facets.pool.query(`NOTIFY ${channel}, '${msg}'`)
```

Comment:
Would be nice to escape `channel` and `msg` to keep SQLi far-off.  Could we use something like this?
```ts
await sql`NOTIFY ${ref(channel)}, ${msg}`.execute(this.db)
```

Reply:
I don't think kysely supports NOTIFY so I can't use those unfortunately :/

However I actually don't need the payload anywhere, so I took that out & limited the channels you can send on which keeps us clear of any danger

Reply:
Ah, nice!  I didn't think kysely had to explicitly support it in order to be able use it with the raw sql builder, but I also haven't tried this out!

Reply:
yeah not sure. i tried it in a few different shapes and couldn't get it to like it. so i just did it this way. 

If we find we need payloads in the future, we can reassess it